xbmc/video/VideoInfoScanner.cpp

```diff
@@ -993,26 +992,13 @@ CVideoInfoScanner::~CVideoInfoScanner()
       int movieId{-1};
       item.SetProperty("from_nfo", true);
 
-      if (existingMovieId > -1)
-      {
-        // Add as version
-        movieId = existingMovieId;
-        item.SetProperty("idMovie", movieId);
-        const int versionId{static_cast<int>(AddVideo(&item, nullptr, bDirNames, true, nullptr,
-                                                      false, ContentType::MOVIE_VERSIONS))};
-        if (versionId < 0)
+      if (UpdateSetInTag(*pItem->GetVideoInfoTag()))
+        if (!AddSet(pItem->GetVideoInfoTag()->m_set))
```

Comment:
@78andyp:  SonarQube finding: Due to this change, previously calculated `existingMovieId` is not used anymore. Thus, that calculation is pointless now and should be removed.

<img width="782" height="334" alt="image" src="https://github.com/user-attachments/assets/cc5bccde-cd62-4954-beef-90f004484660" />